tests/integration/pipeline.ts

```diff
@@ -601,6 +601,17 @@ describe('Pipeline Editor tests', () => {
     cy.findByRole('button', { name: /cancel/i }).click();
   });
 
+  it('airflow pipeline should display error dialog if username and password are not filled in', () => {
+    cy.createPipeline({ type: 'airflow' });
+    cy.savePipeline();
+
+    cy.createRuntimeConfig({ type: 'broken' });
+    cy.get('.jp-Dialog-header').contains('Error making request');
+  
+    // Dismiss dialog
+    cy.findByRole('button', { name: /cancel/i }).click();
```

Comment:
The error dialog only contains the ok button
```suggestion
    cy.findByRole('button', { name: /ok/i }).click();
```